### PR TITLE
Use GitHub Actions to automate tests

### PR DESCRIPTION
**Pull request by parmentf**
As long as automated tests seem to not work any more using travis-ci.org, let's use GitHub Actions, to see if it's convenant.

It might that the reason travis-ci.org is broken for this repository lies into its new header:
image

> Please be aware travis-ci.org will be shutting down in several weeks, with all accounts migrating to travis-ci.com. Please stay tuned here for more information.

### CODE REVIEW

**Comment by parmentf** [comment]
Here is what's happening in all pull requests of ezs:
image

**Comment by parmentf** [comment]
It seems that some checks pass, using travis-ci **.com** (I did not do anything for that).
image

But GitHub still has a pending `continuos-integration/travis-ci` job waiting for status to be reported.
image

**Comment by parmentf** [comment]
What worked once seems to not work any more (since 013f0fb, which was taken on [Coveralls GitHub Action](https://github.com/marketplace/actions/coveralls-github-action).

image

**Comment by parmentf** [comment]
I pushed 0ab666f because of the message:

[image](https://github.com/Inist-CNRS/ezs/actions/runs/664895250)

**Comment by parmentf** [comment]
I did not see the error message beneath: You have an error in your yaml syntax on line 36

**Comment by parmentf** [comment]
One has to let a space between dash and `name`.

**Comment by parmentf** [comment]
It seems to work now (with coveralls).

But there is always one required check that did not report: `continuous-integration/travis-ci`. And I don't know why.
Other travis checks still work (but for how long?), and they take much time.

image

Next step is to remove travis-ci from GitHub repository's checks.
But I fear it is repository's level. That means that we could remove Travis-CI webhook for the repository.
To re-enable checks, we would have to merge this PR to master branch, *and* to refresh all going on PR...

**Comment by parmentf** [comment]
Another point to resolve: this is the simple workflow, using only one version of node.
There is a means to use matrix node-versions, but it requires a parallel use of coveralls (which is [possible](https://github.com/marketplace/actions/coveralls-github-action#complete-parallel-job-example))

**Comment by parmentf** [comment]
All jobs are now working, we even have coveralls comments.
image

But the status is not green because of `continuous-integration/travis-ci` which don't report anything since a long time.

Maybe we shutdown deactivate the Travis-CI webhook, @touv ?
And the quickly merge this PR on master branch.

**Comment by parmentf** [comment]
Oh there is also a Travis-CI App installed at organization level (Inist-CNRS).
Maybe this one still works (it was installed 2 months ago).
I deactivate the old Travis-CI webhook.

**Comment by parmentf** [comment]
Also, before using only GitHub Actions, consider its [pricing](https://docs.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions), which offer 2000 free minutes for Open Source.

**Comment by parmentf** [comment]
I don't understand why, but the Actions minutes seem to be in my personal account (34 minutes), and not in Inist-CNRS's account (0 minutes).

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        #node-version: [10.x, 12.x, 14.x]
+        node-version: [10.x, 14.x]
```
Comment (touv) [inline .github/workflows/node.js.yml:20]: 12.x ????

Comment (parmentf) [reply .github/workflows/node.js.yml:20]: I can put it back. I waas afraid that minutes of Actions were billed. It's not the case.

**Pull request merged**